priv/repo/migrations/20181011082425_update_timestamp_fields.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Hexpm.RepoBase.Migrations.UpdateTimestampFields do
+  use Ecto.Migration
+
+  def change do
+    alter table(:emails) do
+      modify(:verification_expiry, :utc_datetime_usec)
```

Comment:
@ericmj do you have a strong opinion whether this should have or don't have usec? What happened was we previously had it as non-usec and were filling it in with `DateTime.utc_now()` (which as usec), and with Ecto 3.0 we're stricter with regards to precision. Alternatively, we could truncate it before saving into the DB, but because we use usec everywhere else I think it's better to be consistent.